pkgs/top-level/aliases.nix

```diff
@@ -1385,6 +1384,13 @@ mapAliases {
   source-han-serif-korean = throw "'source-han-serif-korean' has been renamed to/replaced by 'source-han-serif'"; # Converted to throw 2025-10-27
   source-han-serif-simplified-chinese = throw "'source-han-serif-simplified-chinese' has been renamed to/replaced by 'source-han-serif'"; # Converted to throw 2025-10-27
   source-han-serif-traditional-chinese = throw "'source-han-serif-traditional-chinese' has been renamed to/replaced by 'source-han-serif'"; # Converted to throw 2025-10-27
+  sourceHanPackages.mono = source-han-mono; # Added 2025-11-03
+  sourceHanPackages.sans = source-han-sans; # Added 2025-11-03
+  sourceHanPackages.sans-vf-otf = source-han-sans-vf-otf; # Added 2025-11-03
+  sourceHanPackages.sans-vf-ttf = source-han-sans-vf-ttf; # Added 2025-11-03
+  sourceHanPackages.serif = source-han-serif; # Added 2025-11-03
+  sourceHanPackages.serif-vf-otf = source-han-serif-vf-otf; # Added 2025-11-03
+  sourceHanPackages.serif-vf-ttf = source-han-serif-vf-ttf; # Added 2025-11-03
```

Comment:
Since we recently reworked all the aliases, please add warnings here as well, with, I think, `warnAlias` or something (defined at the top of the file).

Reply:
added the warnings